docs/howtos/layers/shapes.md

```diff
@@ -140,12 +140,38 @@ centered on that click.
 `Polygons` can be added using an array of N vertices. Polygons are closed by
 default, and so you don't also need to include the first point at the end of the
 array. The order of the vertices will determine the triangulation of the
-polygon, which can be non-convex, but cannot have holes. The polygon creation
-tool can be selected from the layer control panel or by pressing the `P` key
-when the shapes layer is selected. When adding a polygon each click will add a
-vertex at the clicked location. To finish drawing a polygon you must click the
-`escape` key, which will add a final vertex at the current mouse position and
-complete the polygon. You'll then be able to start adding another one.
+polygon, which can be non-convex, but cannot have holes. For drawing polygons, 
+multiple tools can be used.
+The polygon creation tool can be selected from the layer control panel or by 
+pressing the `P` key when the shapes layer is selected. When adding a polygon 
+each click will add a vertex at the clicked location. To finish drawing a polygon 
+you must click the`escape` key, which will add a final vertex at the current mouse 
+position and complete the polygon. You'll then be able to start adding another one.
+
+`Polygons` can also be created with the polygon lasso creation tool. This can be found
```

Comment:
```suggestion
`Polygons` can also be created with the polygon lasso creation tool, which can be found
```

Reply:
Originally I had it the same, but wasn't certain about creating longer sentences.